code/obj/artifacts/artifact_items/agitator.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/artifact/agitator
+	name = "artifact agitator"
+	associated_datum = /datum/artifact/agitator
+
+	afterattack(atom/target, mob/user, reach, params)
```

Comment:
This should ideally be implemented on the artifact datum itself rather than down here, I know artifact datums don't have a direct replacement for `afterattack` but you could add one 🥺

Reply:
Maybe in another PR? Since this isn't the only case where afterattack() is implemented on a handheld?